python-sdk/src/astro/databases/base.py

```diff
@@ -458,6 +458,15 @@ def get_dataframe_from_file(file: File):
 
         return file.export_to_dataframe()
 
+    @staticmethod
+    def _assert_not_empty_df(df):
```

Comment:
Shouldn't this method be overridden for PostGres and Bigquery like you had mentioned?